app/models/install.rb

```diff
@@ -0,0 +1,6 @@
+class Install < ApplicationRecord
```

Comment:
This class looks like it was accidentally created. Did you make this in addition to the User class by accident or am I missing something maybe?